services/types.go

```diff
@@ -1,7 +1,16 @@
 package services
 
+type Role string
```

Comment:
This is a very generic name and might create collision or confusion. 
Please rename to something more meaningful and more specific. 
```suggestion
type OnboardingSubjectRole string
```